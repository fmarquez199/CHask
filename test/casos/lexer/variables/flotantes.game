Skill fl1 = 2'5
Skill fl2 = 2'000
Skill fl3 = 02'000
Skill fl4 = 02

