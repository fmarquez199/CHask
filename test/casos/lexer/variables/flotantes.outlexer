(Skill), pos (1,1)
Identificador "fl1", pos (1,7)
(=), pos (1,11)
Flotante 2'5, pos (1,13)
(\n), pos (1,16)
(Skill), pos (2,1)
Identificador "fl2", pos (2,7)
(=), pos (2,11)
Flotante 2'000, pos (2,13)
(\n), pos (2,18)
(Skill), pos (3,1)
Identificador "fl3", pos (3,7)
(=), pos (3,11)
Flotante 02'000, pos (3,13)
(\n), pos (3,19)
(Skill), pos (4,1)
Identificador "fl4", pos (4,7)
(=), pos (4,11)
Entero 02, pos (4,13)
(\n), pos (4,15)
