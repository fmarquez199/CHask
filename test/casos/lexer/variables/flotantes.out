Token Skill en la fila: 1, columna: 1
Token identificador "fl1" en la fila: 1, columna: 7
Token = en la fila: 1, columna: 11
Token flotante 2'5 en la fila: 1, columna: 13
Token End Instruction \n en la fila: 1, columna: 16
Token Skill en la fila: 2, columna: 1
Token identificador "fl2" en la fila: 2, columna: 7
Token = en la fila: 2, columna: 11
Token flotante 2'000 en la fila: 2, columna: 13
Token End Instruction \n en la fila: 2, columna: 18
Token Skill en la fila: 3, columna: 1
Token identificador "fl3" en la fila: 3, columna: 7
Token = en la fila: 3, columna: 11
Token flotante 02'000 en la fila: 3, columna: 13
Token End Instruction \n en la fila: 3, columna: 19
Token Skill en la fila: 4, columna: 1
Token identificador "fl4" en la fila: 4, columna: 7
Token = en la fila: 4, columna: 11
Token entero 02 en la fila: 4, columna: 13
Token End Instruction \n en la fila: 4, columna: 15
Token End Instruction \n en la fila: 5, columna: 1
